pkg/cover/instrument.go

```diff
@@ -260,8 +260,12 @@ func listen() (ln net.Listener, host string, err error) {
 			return
 		}
 	}
-
-	ln, err = net.Listen("tcp4", ":0")
+	agentPort := "{{.AgentPort }}"
+	if agentPort != "" {
+		ln, err = net.Listen("tcp4", agentPort)
+	}else {
```

Comment:
format

Reply:
format is fine, there seems to be extra space on the web page.